modules/nf-core/bedtools/map/meta.yml

```diff
@@ -0,0 +1,53 @@
+name: bedtools_map
+description: Allows one to screen for overlaps between two sets of genomic features.
+keywords:
+  - bed
+  - map
+tools:
+  - bedtools:
+      description: |
+        A set of tools for genomic analysis tasks, specifically enabling genome arithmetic (merge, count, complement) on various file types.
+      documentation: https://bedtools.readthedocs.io/en/latest/content/tools/map.html
+      licence: ["MIT"]
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - intervals1:
+      type: file
+      description: BAM/BED/GFF/VCF
+      pattern: "*.{bam|bed|gff|vcf}"
+  - intervals2:
+      type: file
+      description: BAM/BED/GFF/VCF
+      pattern: "*.{bam|bed|gff|vcf}"
+  - meta2:
+      type: map
+      description: |
+        Groovy Map containing reference chromosome sizes
+        e.g. [ id:'test' ]
+  - chrom_sizes:
+      type: optional file
```

Comment:
```suggestion
      type: file
```

Reply:
Why chrom_sizes is not optional?

Reply:
My bad I should have explained here a bit. The types are for example `file`, 'string`, `integer` etc. The fact that it's optional should be in the description :)